client/my-sites/themes/theme-options.js

```diff
@@ -87,8 +100,17 @@ const tryandcustomize = {
 		comment: 'label in the dialog for opening the Customizer with the theme in preview'
 	} ),
 	action: tryAndCustomizeAction,
-	hideForSite: ( state, siteId ) => ! canCurrentUser( state, siteId, 'edit_theme_options' ),
-	hideForTheme: ( state, theme, siteId ) => isThemeActive( state, theme.id, siteId )
+	hideForSite: ( state, siteId ) => (
+		! canCurrentUser( state, siteId, 'edit_theme_options' ) ||
+		( isJetpackSite( state, siteId ) && isJetpackSiteMultiSite( state, siteId ) )
+	),
+	hideForTheme: ( state, theme, siteId ) => (
+		isThemeActive( state, theme.id, siteId ) ||
+		( isJetpackSite( state, siteId ) && isThemePremium( state, theme.id ) ) ||
```

Comment:
This blocks _try&customize_ for premium themes on AT sites. I think this logic should match _activate_?